app/models/site_config.rb

```diff
@@ -36,4 +36,8 @@ class SiteConfig < RailsSettings::Base
   # Email digest frequency
   field :periodic_email_digest_max, type: :integer, default: 0
   field :periodic_email_digest_min, type: :integer, default: 2
+
+  # Tags
+
+  field :suggested_tags, type: :array, default: %w[beginners career computerscience javascript security ruby rails swift kotlin]
```

Comment:
I saw you removed a lot of the tags in the previous `Tag::NAMES`...

Reply:
Yeah, this is the new default in development. I figured it could be a simpler list because we have a shorter list of tags used by the default generated seed than production which uses all the tags.